src/Evolution/Systems/GrMhd/ValenciaDivClean/BoundaryConditions/FreeOutflow.cpp

```diff
@@ -101,6 +105,12 @@ std::optional<std::string> FreeOutflow::dg_ghost(
     }
   }
 
+  determinant_and_inverse(make_not_null(&interior_sqrt_det_spatial_metric),
```

Comment:
Maybe add a comment as to why this isn't just retrieved? I think I commented on the original PR asking about it :P